unit/java_bytecode/generate_concrete_generic_type/generic_two_fields.java

```diff
@@ -0,0 +1,29 @@
+public class generic_two_fields {
+
+  // For this to work we need to compile with -g, otherwise the symbols we are looking for 
+  // are entering the symbol table as anonlocal::1 and anonlocal::2 and nothing works.  
+  class bound_element<NUM extends java.lang.Number> {
+    NUM first;
+    NUM second;
+  }
+
+  bound_element<Integer> belem;
+
+  Integer f(int n) {
+    bound_element<Integer> e=new bound_element<Integer>();
```

Comment:
To avoid making this test unnecesarily confusing, please remove irrelevant code (e.g. maybe this entire function?)